components/general/FeaturedWorkSection.tsx

```diff
@@ -0,0 +1,58 @@
+import Picturedbox from "../base/Picturedbox";
+import styles from "../../styles/FeaturedWork.module.css";
+import { textFont, titleFont } from "../../pages/_app";
+
+export default function Home() {
+  return (
+    <div className={styles["featured-work"]}>
+      <div className={styles["featured-work__left-container"]}>
+        <h1 className={`${titleFont.className} global__heading-h1 `}>
+          FEATURED <br /> WORK
+        </h1>
+        <p className={`${textFont.className} global__text--large`}>
+          The featured work section shows some of the successful works of our website.
+        </p>
+        <button className={styles["featured-work__btn"]}>SEE ALL WORK</button>
```

Comment:
Use the Button component instead of button tag

Reply:
fixed, thks